server/game/cards/13.3-PoS/MelleonsTome.js

```diff
@@ -0,0 +1,39 @@
+const DrawCard = require('../../drawcard.js');
+const GameActions = require('../../GameActions');
+
+class MalleonsTome extends DrawCard {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: ability.effects.addIcon('intrigue')
+        });
+
+        this.reaction({
+            when: {
+                afterChallenge: event =>
+                    event.challenge.attackingPlayer === this.controller &&
+                    event.challenge.winner === this.controller &&
+                    this.parent.isAttacking()
```

Comment:
Might need an explicit `this.parent &&` here, since it only should trigger when attached.

(Unfortunately can't auto-check easily because things like Hotah's Axe have abilities that trigger while not attached)
 ![](http://lcg-cdn.fantasyflightgames.com/got2nd/GT30_19.jpg)